src/Geometry.cpp

```diff
@@ -739,27 +739,17 @@ void Geometry::subdivideCells() {
 
 /**
  * @brief Compute the number of flat source regions in the Geometry and
```

Comment:
Looks like this comment wasn't updated from PR #98 - this routine does not calculate the number of FSRs any longer. I know this isn't related to this PR, but let's go ahead and fix it since it is front and center.